src/Backend/Modules/Pages/Domain/Page/Page.php

```diff
@@ -242,7 +242,10 @@ public function __construct(
         $this->navigationTitleOverwrite = $navigationTitleOverwrite;
         $this->hidden = $hidden;
         $this->status = $status;
-        $this->data = serialize($data);
+        $this->data = $data;
+        if ($data !== null) {
```

Comment:
I would use the life cycle events for this

Reply:
Changed in 01db59a0d8b3732875502fe3f6407d2738e25554